src/main/main.ts

```diff
@@ -97,10 +97,11 @@ const createWindow = async () => {
 
   const view = new BrowserView({
     webPreferences: {
-      webviewTag: true,
-      preload: `${path.join(__dirname, './preload.js')}`,
+      // preload: resolvePreloadPath(),
+      preload: `${__dirname}/preload.js`,
```

Comment:
It's not resolved completely at the moment, preload should be actually included as `<script>` tab, additionally to the line above, which is impossible to do for remove urls. But hopefully, we will remove this part of the app. Correct me if I'm wrong @tem-tem